include/brickengine/std/sign.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef FILE_SIGN_HPP
+#define FILE_SIGN_HPP
+
+template <typename T> int sign(T val) {
+    return (T(0) < val) - (val < T(0));
```

Comment:
I would add a comment what this does. I reckon it is the following but would go in a little more detail; https://stackoverflow.com/questions/1903954/is-there-a-standard-sign-function-signum-sgn-in-c-c

Reply:
https://en.wikipedia.org/wiki/Sign_function